### ISSUE TITLE
not warns when invalid property

### ISSUE THREAD

**Issue by GitScrum** [stylelint/stylelint#935]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug where not reported a bug

> Which rule, if any, is this issue related to?

`property-not-declared` (maybe feature?)

> What CSS is needed to reproduce this issue?

```css
.test {
  rgin-top: 5px;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "extends": "stylelint-config-standard"
}
```

> Which version of stylelint are you using?

`5.0.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

"PostCSS plugin"

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

"Warnings"

> What actually happened (e.g. what warnings or errors you are getting)?

"errors"

**Comment by davidtheclark** [comment]
I'm afraid it's currently out of stylelint's scope to check the validity of all the identifier and keywords, property-value pairs, etc., that you use. 

We could feasibly create a rule using https://github.com/twolfson/css-validator or something like that which would run your code against the old CSS validator --- but most people are using some form of processing that would trigger false positives in that system.

A project just started recently at https://github.com/ben-eb/css-values whose aim is to provide the low-level tools that projects like stylelint would need to verify validity. Once that takes off we'll be able to consider a rule like this.
